src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -40,6 +38,40 @@ elseif($ENV{__BuildArch} STREQUAL arm OR $ENV{__BuildArch} STREQUAL arm64)
     )
 endif()
 
+if($ENV{__appContainer} STREQUAL true)
+    set (NATIVECOMPRESSION_SOURCES
+        ${NATIVECOMPRESSION_SOURCES}
+        clrcompression.AppX.def
```

Comment:
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#code-file-naming-conventions says to use Uap for Uap specific stuff.